docs/manage/ingestion-volume/ingest-budgets/index.md

```diff
@@ -44,11 +44,13 @@ There are two versions of ingest budgets:
 * V1 ingest budgets are older and have a **Field Value** for Collector assignment. They are shown with a **V1** in the **Name** cell.<br/> ![v1 budget tag.png](/img/ingestion-volume/v1-budget-tag.png)
 * V2 ingest budgets provide you the ability to assign budgets to your log data by either [Fields](/docs/manage/fields) or the following [built in metadata](/docs/search/get-started-with-search/search-basics/built-in-metadata) fields, `_collector`, `_source`, `_sourceCategory`, `_sourceHost`, and `_sourceName`.
 
-:::important
-You can edit an existing **V1** budget by providing a **scope** to change it to a **V2** budget that is metadata based. V2 budgets are manageable with the Ingest Budget Management API **V2**.
-:::
 
-Once a **V1** budget is given a **scope** and changed to **V2** it is permanent and can not be reversed.
+## Migrating V1 budgets to V2
+
+* You can edit an existing **V1** budget by providing a **scope** to change it to a **V2** budget that is metadata-based.
+* V2 budgets are manageable with the [Ingest Budget Management **V2** API](/docs/api/ingest-budget-v2).
+* V1 budgets will be deprecated on September 16, 2024.
```

Comment:
@ajaiswals will all users need to migrate from v1 to v2 by this date?

Reply:
@kimsauce - yes and we are starting to sending notifications to customer that we have V1 already.

Reply:
Is this related to [Ingest Budgets for OT collector](https://drive.google.com/file/d/1lBeb3Y8DVtY7tudKhQBV-1WwsUtRcs2w/view)? @ajaiswals